src/libs/help.js

```diff
@@ -19,7 +19,7 @@ function getGreaterLength(currentMax, itemToCheck) {
 
 /**
  * @param {object} args
- * @param {import(".").NS} args.ns Use just "@param {NS} ns" if editing in game
+ * @param {import("..").NS} args.ns Use just "@param {NS} ns" if editing in game
```

Comment:
Is this supposed to be two dots?

Reply:
In this case yes. The import is a relative path. So getting the intellisense to work for the `ns` variable requires us to look up one directory in the `src` folder. Not in the `src/libs` folder.